packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -1,207 +1,207 @@
   },
   {
     "chunkShortName": "headline",
     "size": 4981,
-    "gzipSize": 2108
+    "gzipSize": 2106
   },
   {
     "chunkShortName": "icon",
-    "size": 14976,
+    "size": 14979,
     "gzipSize": 7480
   },
   {
     "chunkShortName": "inline-notification",
-    "size": 4508,
-    "gzipSize": 1973
+    "size": 4509,
+    "gzipSize": 1977
   },
   {
     "chunkShortName": "link",
-    "size": 6245,
-    "gzipSize": 2570
+    "size": 6251,
+    "gzipSize": 2572
   },
   {
     "chunkShortName": "link-pure",
-    "size": 6634,
-    "gzipSize": 2812
+    "size": 6641,
+    "gzipSize": 2815
   },
   {
     "chunkShortName": "link-social",
-    "size": 6991,
-    "gzipSize": 2436
+    "size": 6996,
+    "gzipSize": 2441
   },
   {
     "chunkShortName": "link-tile",
-    "size": 6875,
-    "gzipSize": 2903
+    "size": 6881,
+    "gzipSize": 2906
   },
   {
     "chunkShortName": "main",
-    "size": 66493,
-    "gzipSize": 22725
+    "size": 67369,
+    "gzipSize": 22905
   },
   {
     "chunkShortName": "marque",
-    "size": 4738,
-    "gzipSize": 2001
+    "size": 4740,
+    "gzipSize": 2002
   },
   {
     "chunkShortName": "modal",
-    "size": 7993,
-    "gzipSize": 3435
+    "size": 7992,
+    "gzipSize": 3431
   },
   {
     "chunkShortName": "pagination",
-    "size": 5925,
-    "gzipSize": 2479
+    "size": 5927,
+    "gzipSize": 2480
   },
   {
     "chunkShortName": "popover",
-    "size": 6692,
-    "gzipSize": 2801
+    "size": 6696,
+    "gzipSize": 2806
   },
   {
     "chunkShortName": "radio-button-wrapper",
-    "size": 7835,
-    "gzipSize": 3296
+    "size": 7826,
+    "gzipSize": 3282
   },
   {
     "chunkShortName": "scroller",
-    "size": 5506,
-    "gzipSize": 2245
+    "size": 5508,
+    "gzipSize": 2246
   },
   {
     "chunkShortName": "segmented-control",
-    "size": 6970,
+    "size": 6973,
     "gzipSize": 2776
   },
   {
     "chunkShortName": "select-wrapper",
-    "size": 22871,
+    "size": 22884,
     "gzipSize": 7531
   },
   {
     "chunkShortName": "spinner",
-    "size": 2431,
-    "gzipSize": 1216
+    "size": 2433,
+    "gzipSize": 1217
   },
   {
     "chunkShortName": "stepper-horizontal",
-    "size": 11514,
-    "gzipSize": 4747
+    "size": 11522,
+    "gzipSize": 4750
   },
   {
     "chunkShortName": "switch",
-    "size": 5304,
-    "gzipSize": 2195
+    "size": 5301,
+    "gzipSize": 2191
   },
   {
     "chunkShortName": "table",
-    "size": 10456,
-    "gzipSize": 3860
+    "size": 10463,
+    "gzipSize": 3856
   },
   {
     "chunkShortName": "tabs",
-    "size": 6135,
-    "gzipSize": 2594
+    "size": 6141,
+    "gzipSize": 2595
   },
   {
     "chunkShortName": "tabs-bar",
-    "size": 8300,
+    "size": 8307,
     "gzipSize": 3303
   },
   {
     "chunkShortName": "tag",
-    "size": 3912,
-    "gzipSize": 1720
+    "size": 4684,
+    "gzipSize": 1857
   },
   {
     "chunkShortName": "tag-dismissible",
-    "size": 5193,
-    "gzipSize": 2134
+    "size": 4610,
```

Comment:
FYI: tag-dismissible chunk size decreased because I moved some code from `tag-styles.ts` into `tag-shared-utils.ts`, so that changing `tag-styles.ts` doesn't affect the "tag-dismissible" chunk size